src/dymaptic.GeoBlazor.Core/Components/Layers/ImageryLayer.cs

```diff
@@ -0,0 +1,143 @@
+﻿using dymaptic.GeoBlazor.Core.Components.Renderers;
+using dymaptic.GeoBlazor.Core.Exceptions;
+using dymaptic.GeoBlazor.Core.Serialization;
+using Microsoft.AspNetCore.Components;
+using System.Text.Json.Serialization;
+
+
+namespace dymaptic.GeoBlazor.Core.Components.Layers;
+
+/// <summary>
+///     Represents a dynamic image service resource as a layer. An ImageryLayer retrieves and displays data from dynamic image services. An image service supports
+///     accessing the mosaicked image, its catalog, and the individual rasters in the catalog. An image service supports dynamic access and tiled access. 
+///     <a target="_blank" href="https://developers.arcgis.com/javascript/latest/api-reference/esri-layers-ImageryLayer.html">
+///         ArcGIS
+///         JS API
+///     </a>
+/// </summary>
+public class ImageryLayer : Layer
+{
+    /// <inheritdoc />
+    [JsonPropertyName("type")]
+    public override string LayerType => "imagery";
+
+    /// <summary>
+    ///     Parameterless constructor for use as a razor component
+    /// </summary>
+    public ImageryLayer()
+    {
+    }
+
+    /// <summary>
+    ///     Constructor for use in code
+    /// </summary>
+    /// <param name="url"></param>  
+    /// <param name="portalItem"></param>
+    /// <param name="renderer"></param>
+    /// <param name="format"></param>
+    public ImageryLayer(string? url = null, PortalItem? portalItem = null, IImageryRenderer? renderer = null
+        ) 
+    {
+        if (url is null && portalItem is null)
+        {
+            throw new MissingRequiredOptionsChildElementException(nameof(ImageryLayer),
+                new[] { nameof(Url), nameof(PortalItem) });
+        }
+        Renderer = renderer;
+        Url = url;
+        PortalItem = portalItem;
+
+    }
+
+    /// <summary>
+    ///     The url for the Imagery Layer source data.
+    /// </summary>
+    [Parameter]
+    [JsonIgnore(Condition = JsonIgnoreCondition.WhenWritingNull)]
+    [RequiredProperty(nameof(PortalItem))]
+    public string? Url { get; set; }
+
+    /// <summary>
+    ///     The portal item for the Imagery Layer source data.
+    /// </summary>
+    [RequiredProperty(nameof(Url))]
+    public PortalItem? PortalItem { get; set; }
+
+    /// <summary>
+    ///     An interface that implements the various imagery renderers.
+    /// </summary>
+    public IImageryRenderer? Renderer { get; set; }
+
+    /// <inheritdoc />
+    public override async Task RegisterChildComponent(MapComponent child)
+    {
+        switch (child)
+        {
+            case PortalItem portalItem:
+                if (!portalItem.Equals(PortalItem))
+                {
+                    PortalItem = portalItem;
+                    LayerChanged = true;
+                }
+
+                break;
+            //case IImageryRenderer renderer:
```

Comment:
What happened with the renderer? We're not supporting it? I think you were working on implementing it in GB.Pro, but this code would still need to be uncommented.

Reply:
turned the renderer back on.  I think I had shut it off while troubleshooting an item.